File: solutions/python/resistor-color-duo/1/resistor_color_duo.py

_COLORS = {
        "black" : 0,
        "brown" : 1,
        "red" : 2,
        "orange" : 3,
        "yellow" : 4,
        "green" : 5,
        "blue" : 6,
        "violet" : 7,
        "grey" : 8,
        "white" : 9
    }


def value(colors:list) -> int:
    """
    Returning type of two rasberry pi resistors
    """

    resistor1 = _COLORS[colors[0]]
    resistor2 = _COLORS[colors[1]]

    return int(str(resistor1) + str(resistor2))

File: solutions/csharp/resistor-color-duo/1/ResistorColorDuo.cs
public static class ResistorColorDuo
{
    public static int Value(string[] colors)
    {
        string resistorValue = "";

        foreach(string color in colors)
        {
            string colorValue = ColorResistor(color);
            resistorValue += colorValue;
        };

        string finalResistorValue = resistorValue[..2];

        return int.Parse(finalResistorValue);

        
    }

    private static string ColorResistor(string color) => color switch 
    {
            "black" => "0",
            "brown" => "1",
            "red" => "2",
            "orange" => "3",
            "yellow" => "4",
            "green" => "5",
            "blue" => "6",
            "violet" => "7",
            "grey" => "8",
            "white" => "9",
            
    };

    
}
